### ISSUE TITLE
Support adding types for boot files (Axios, Vuex, etc)

### ISSUE THREAD

**Issue by IlCallo** [quasarframework/app-extension-typescript#29]
As seen [here](https://github.com/quasarframework/app-extension-typescript/pull/22#issuecomment-513350018), the AE should also check which boot files have been added from the Quasar CLI and add typings to those one too.

**Comment by IlCallo** [comment]
Example for i18n boot file

```ts
import messages from 'src/i18n';
import Vue, { VueConstructor } from 'vue';
import VueI18n from 'vue-i18n';

declare module 'vue/types/vue' {
  interface Vue {
    i18n: VueI18n;
  }
}

export default async ({ app, Vue }: { app: Vue; Vue: VueConstructor }) => {
  Vue.use(VueI18n);

  // Set i18n instance on app
  app.i18n = new VueI18n({
    locale: 'en-us',
    fallbackLocale: 'en-us',
    messages,
  });
};
```

**Comment by razbakov** [comment]
After solving #31 if I run `quasar dev` I am getting

```
ERROR in src/boot/axios.ts(3,25):
TS7031: Binding element 'Vue' implicitly has an 'any' type.
ERROR in src/boot/i18n.ts(13,19):
TS7031: Binding element 'app' implicitly has an 'any' type.
ERROR in src/store/index.ts(21,5):
TS2322: Type 'string | undefined' is not assignable to type 'boolean | undefined'.
  Type 'string' is not assignable to type 'boolean | undefined'.
Version: typescript 3.5.3
```

**Comment by eyedean** [comment]
... and it really hurts!  It's been half a day so far for me trying to get Typescript on Quasar (SPA + SASS + Axios + i18n) set up and I am constantly running into issues like ☝️  with absolutely no example to follow.

**Comment by nothingismagick** [comment]
Sorry you're having difficulties here. We're all volunteers working
together to try and move things forward, so please bear with us - we're
definitely NOT trying to hurt anybody!

We know that there needs to be better documentation, and that is coming
soon.

**Comment by outofmemoryagain** [comment]
Also, until we gets the docs up to speed, we are pretty responsive to questions posted on the [Quasar Discord Server](http://chat.quasar.dev) typescript channel.

**Comment by paulvanbladel** [comment]
I'm solving it as follows:
```
export default async ({ Vue }: { Vue: any }) => {
  Vue.prototype.$axios = axios
}
```

**Comment by IlCallo** [comment]
Should be fixed by #48
